### PR TITLE
doc(sandbox): add more instructions for starting on sandbox work

### PR DESCRIPTION
**Pull request by yufansong**
As title, add some instructions when start working on sandbox. Most instructions are the problem I meet when begenning.

### CODE REVIEW

File: opendevin/requirement.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+docker
```
Comment (rbren) [inline opendevin/requirement.txt:1]: I think these are already in `/requirements.txt`, no?

Comment (yufansong) [reply opendevin/requirement.txt:1]: Oh, sorry, just find the we already have a `requirement.txt` in root folder. Maybe I forget to update the codebase when dev. Will update this PR.

Comment (yufansong) [reply opendevin/requirement.txt]: Removed.

File: opendevin/README.md [hunk 2]
```diff
@@ -16,3 +16,27 @@ It will map `./workspace` into the docker container with the folder permission c
...
+
+3. Install the requirement package.
+```bash
+pip install -r requirement.txt                                                                 
+```
+If you still meet problem like `ModuleNotFoundError: No module named 'agenthub'`, try to add the `opendevin` root path into `PATH` env.
+
+4. Run following cmd to start.
+```bash
+PYTHONPATH=pwd python ./opendevin/main.py -d ./workspace -t "write a bash script that prints hello world"
```
Comment (rbren) [inline opendevin/README.md:41]:
```suggestion
PYTHONPATH=`pwd` python ./opendevin/main.py -d ./workspace -t "write a bash script that prints hello world"
```

**Review by rbren — APPROVED** [review]

**Pull request merged**